client/state/sharing/publicize/test/reducer.js

```diff
@@ -25,30 +29,36 @@ import { useSandbox } from 'test/helpers/use-sinon';
 describe( 'reducer', () => {
 	describe( '#fetchingConnections()', () => {
 		it( 'should set fetching to true for fetching action', () => {
-			const state = fetchingConnections( null, {
-				type: PUBLICIZE_CONNECTIONS_REQUEST,
-				siteId: 2916284
-			} );
+			[ PUBLICIZE_CONNECTION_REQUEST, PUBLICIZE_CONNECTIONS_REQUEST ].map( ( type ) => {
```

Comment:
Also, this highlights a reason for normalizing to a single action creator so that you don't need to account for and test multiple action types. See `receivePost` as an example:

https://github.com/Automattic/wp-calypso/blob/4cc87cb/client/state/posts/actions.js#L33-L42